pymisp/aping.py

```diff
@@ -106,8 +106,7 @@ def remote_acl(self, debug_type: str='findMissingFunctionNames'):
     @property
     def describe_types_local(self):
         '''Returns the content of describe types from the package'''
-        with (self.resources_path / 'describeTypes.json').open() as f:
-            describe_types = json.load(f)
+        describe_types = self._load_json(str(self.resources_path / 'describeTypes.json'))
```

Comment:
I think this could also be changed to use the descripe_types variable from abstract.py by importing the loaded content.

Reply:
:+1: